.travis.yml

```diff
@@ -4,22 +4,28 @@ sudo: required
 dist: precise
 language: python
 python:
-  - 2.7
+  - "2.7"
+  - "3.6"
```

Comment:
Any particular reason why Python 3 CI check is needed in this PR?

The reason I'm asking: adding more tasks into TravisCI will slow down the entire build time, since more jobs will go into the qeue (we verified that before). In short: we're using max tasks Travis gives us for free OSS, so there should be a good reason before we'll add new tasks into TravisCI.
So far as I know we don't have any plans about re-working entire platform for py3 support, but there could be smaller features like py3 runners in future.

Reply:
I do believe it would be good to eventually also runs tests on Python 3, but @armab has valid point so it's better comment it out (for now).

Reply:
Nope, I'm playing around with Travis, but I realized that I'm playing around with Travis on this branch. I'll stop that (and fix this).

Reply:
If we can put that effort to run tests in Py3 and work on running StackStorm in py3 env, - I'm all  :100: for that to happen, but -1 if we just run the failing tests wasting the resources.

Anyways, if anyone wants to play with Python 3, - that would be a really good experiment as part of another PR :+1:

Reply:
@armab Yeah, I wrote this PR, pushed this branch, and then forgot to switch branches when I started playing with Python 3 tests. Derp. 😆

I'll remove all of the Python 3 references, so this PR will only target the documentation as expected, and I'll put the Python 3 stuff into a different PR.